lib/ZwaveClient.js

```diff
@@ -521,11 +522,20 @@ function valueChanged (nodeid, comclass, valueId) {
     }
     // update cache
     ozwnode.values[value_id] = valueId
+
+    // update last active timestamp
+    ozwnode.lastActive = Date.now()
+    triggerStatusUpdate = true
   }
 
   // check if node is added as secure node
   if (comclass === 0x98 && valueId.index === 0) {
     ozwnode.secure = valueId.value
+    triggerStatusUpdate = true
+  }
+
+  // trigger a status update if something important has changed
+  if (triggerStatusUpdate) {
```

Comment:
I'm not sure I want this, because this could cause some unexpected behaviour in the ui. If you have a selected node in the table every time a value of that node changes this is called and the ui will need to refresh all the tab content. IMO the triggerstatusupdate is useless and you could just leave there the lastactive update

Reply:
Ok I'll remove the triggerStatusUpdate stuff.
So you mean that I shouldn't call `this.emit('nodeStatus', ozwnode)` at all after changing `ozwnode.lastActive`, right?